src/server/game/Entities/Player/PlayerUpdates.cpp

```diff
@@ -426,6 +423,14 @@ void Player::Update(uint32 p_time)
     }
 }
 
+void Player::Heartbeat()
+{
+    Unit::Heartbeat();
+
```

Comment:
1. I put a LOG_ERROR("", "1"); here. 
2. My two char joined the group. But the log only shows once. 
3. The 5-minute log only showed once.